numpy/core/src/umath/ufunc_object.c

```diff
@@ -1025,6 +1029,13 @@ get_ufunc_arguments(PyUFuncObject *ufunc,
             }
 
             switch (str[0]) {
+                case 'a':
```

Comment:
Is there a reason we're not using PyArg_ParseTupleAndKeywords here? Out of scope for this PR, but still

Reply:
We didn't add the comment, but I'm not going to demand it before merging.

Reply:
I think a comment on the structure is a bit out-of-scope of this PR (and I don't really know what the original reason was anyway...)